openfisca_us/variables/gov/irs/taxable_income/deductions/tax_unit_itemizes.py

```diff
@@ -0,0 +1,19 @@
+from openfisca_us.model_api import *
+
+
+class tax_unit_itemizes(Variable):
```

Comment:
Do we need this variable? Can't `taxable_income_deductions` just be `max_(taxable_income_deductions_if_itemizing, taxable_income_deductions_if_not_itemizing)`?

Reply:
I think it's probably more useful than not (e.g. a reform can easily abolish itemization by neutralising it)